src/battle/area_arn/4FF0D0.c

```diff
@@ -2,4 +2,24 @@
 
 #define NAMESPACE b_area_arn
 
-INCLUDE_ASM(s32, "battle/area_arn/4FF0D0", func_80218680_4FF0D0);
+ApiStatus func_80218680_4FF0D0(Evt* script, s32 isInitialCall) {
+    Bytecode *args = script->ptrReadPos;
+    s32 var1;
```

Comment:
TempBlah can be declared here next and initialized on the same line. and then you can initialize vars 1, 2, 3, 4 on the same lines they're declared

Reply:
Will be in the next commit.

Reply:
ah kk, sorry - thought you had intended this to be resolved already